Cron/RetrieveReviews.php

```diff
@@ -50,40 +51,55 @@ public function __construct(
         $this->log = $logger;
         $this->variable = $variable;
 
-        $this->filePath = $this->getFeedUrl();
+        $this->apiKey = $this->getApiKey();
+        $this->locationId = $this->getLocationId();
     }
 
 
-    public function getFeedUrl(){
+    public function getApiKey(){
```

Comment:
Please move this stuff to a new class on `Elgentos\Kiyoh\Model\Config` and implement that in the constructor. That way you can also use the logic in other places.